datascience/tables.py

```diff
@@ -801,6 +801,100 @@ def annotate(axis, ticks):
             height = vargs.pop('height')
         self._visualize(labels, yticks, overlay, draw, annotate, height=height)
 
+    def bar(self, column_for_categories, overlay=False, **vargs):
```

Comment:
There must be a cleaner way to implement bar charts based on existing code for barh. I'll take a look.